sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -18,7 +18,7 @@ categories = ["api-bindings"]
 edition = "2021"
 
 [dependencies]
-azure_core = { path = "../azure_core", default-features = false }
+azure_core = { path = "../azure_core", version = "0.23.0", default-features = false }
```

Comment:
Versions should only be in the workspace `Cargo.toml`. I don't want to manager versions across a swatch of files.

Reply:
But I thought we can put *both* the `path` and `version` in the workspace `Cargo.toml`. I do not want to keep track of all the versions in the crates' `Cargo.toml`s. Most other languages use centralized versions/dependencies as well. It's far easier to keep track of and resolve CG alerts, etc.